chemclipse/plugins/org.eclipse.chemclipse.ux.extension.ui/src/org/eclipse/chemclipse/ux/extension/ui/swt/TaskTile.java

```diff
@@ -254,16 +255,21 @@ private void updateStyle(int style) {
 			textSection.setEnabled(false);
 			textDesciption.setEnabled(false);
 		}
-		//
-		int fontSize;
-		//
+		updateFont(textSection, style);
+	}
+
+	private void updateFont(Label label, int style) {
+
+		int fontSize = 18;
 		if((style & LARGE_TITLE) != 0) {
 			fontSize = 40;
-		} else {
-			fontSize = 18;
 		}
-		//
-		textSection.setFont(Fonts.getCachedFont(getDisplay(), "Arial", fontSize, SWT.BOLD)); //$NON-NLS-1$
+		FontData[] fontDatas = label.getFont().getFontData();
+		for(FontData fontData : fontDatas) {
+			fontData.setHeight(fontSize);
+			fontData.setStyle(SWT.BOLD);
+		}
+		textSection.setFont(new Font(getDisplay(), fontDatas));
```

Comment:
Does the font need to be disposed?
https://help.eclipse.org/latest/index.jsp?topic=%2Forg.eclipse.platform.doc.isv%2Freference%2Fapi%2Forg%2Feclipse%2Fswt%2Fgraphics%2FFont.html